csharp/src/Microsoft.ML.OnnxRuntime/FixedBufferOnnxValue.cs

```diff
@@ -0,0 +1,72 @@
+using Microsoft.ML.OnnxRuntime.Tensors;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    /// <summary>
+    /// Represents an Onnx Value with its underlying buffer pinned
```

Comment:
Should we add a comment about needing to know the exact dimensions of the Tensor (length of the flattened buffer) to use this class ?

Reply:
I agree to add this. do you think where is the best place to add this notes, in FixedBufferOnnxValue's class summary, or CreateFromTensor's summary, or in Run()'s argument?

Reply:
CreateFromTensor's summary ?